drf_extra_fields/fields.py

```diff
@@ -93,7 +93,7 @@ class Base64ImageField(Base64FieldMixin, ImageField):
     INVALID_FILE_MESSAGE = _("Please upload a valid image.")
     INVALID_TYPE_MESSAGE = _("The type of the image couldn't be determined.")
 
-    def get_file_extension(self, filename, decoded_file):
+    def get_file_extension(self, filename, decoded_file, header=None):
         extension = imghdr.what(filename, decoded_file)
```

Comment:
The header argument doesn't do anything. Any example of usecase?

Reply:
`imghdr.what()` does what you want. It get's the extension from the file itself. Which is same with the header. Even if you provide a file name like `images.gif` if the image is a jpg file, the extension will be `jpg`.